libraries/AP_HAL/board/sitl.h

```diff
@@ -1,5 +1,19 @@
 #pragma once
 
+ //### test of conditional compiling
+#ifndef ENABLED
+    #define ENABLED 1
+#endif
+
+#define MISSION_RELATIVE ENABLED
+//#define MISSION_RELATIVE 0
+
+#ifndef SOARING_ENABLED
+    #define SOARING_ENABLED ENABLED
```

Comment:
Enabling soaring on all sitl instances might not be a good idea

Reply:
Tridge told me to do that in sitl.h for my test, but I've been a little insecure if I should let it in for PR. What do you recommend?

Reply:
Oh, now I have seen it - soaring should just be included in planes.